gnowledge/gstudio
# Merging of khaal-dev branch in Mongokit

**Merging of khaal-dev branch in Mongokit:**
- *Khaal-dev*, contains code written specifically for enhancement of course player.
- *Khaal-dev* branch involves major changes in `SAAS/CSS`, `html` and `python` files.

**Following are the major changes/additions in *Khaal-dev* branch:**
- Creation of new SASS profile for **clix** project.
- Addition of newer logo's and images regarding same.
- Quick creation of some of the html and python files from existing files.
- Defining many new additional templates, views and urls.
- Added new configuration variable: `GSTUDIO_SECOND_LEVEL_HEADER`
  - Added with purpose of either showing/hiding second level header. 
  - It accepts `bool` value (`True` / `False`).
  - By default it's `True` (i.e: Second-level header will be visible).
- Freezed version's of all front-end dependencies in `bower.json`.

--

**TEST**
- After taking pull, test the existing features.
- Change the local settings and test course and other major modules.
- Check for breaking of CSS.
